src/execution/sql/table_vector_iterator.cpp

```diff
@@ -23,41 +31,99 @@ bool TableVectorIterator::Init() {
   // Find the table
   table_ = exec_ctx_->GetAccessor()->GetTable(table_oid_);
   TERRIER_ASSERT(table_ != nullptr, "Table must exist!!");
-
+  if (col_oids_.empty()) {
+    table_->GetAllColOid(&col_oids_);
+  }
   // Initialize the projected column
-  TERRIER_ASSERT(!col_oids_.empty(), "There must be at least one col oid!");
+  // TERRIER_ASSERT(!col_oids_.empty(), "There must be at least one col oid!");
```

Comment:
Does this assert fail?

Reply:
Yes, why was this assert disabled? We don't want to do this carelessly.

Reply:
This assertion should never fail because even if the column id vector is empty, it will process all columns instead. But I guess there's no harm to do a double-check here. I'll uncomment it later.